addon_service/common/network.py

```diff
@@ -109,6 +109,10 @@ async def _try_send(self, request: HttpRequestInfo):
             json=request.json,
             data=request.content,
         ) as _response:
+            print(_response)
```

Comment:
I strongly advice against using plain prints in production code, moreover using them to log responses as they may contain sensitive data
```suggestion
            _logger.debug(_response)
```

Reply:
Apologies, I forgot to remove the debugging artifacts.